pscheduler-core/pscheduler-core/bash_completion

```diff
@@ -0,0 +1,158 @@
+# pscheduler(1) completion                                 -*- shell-script -*-
+
+_pscheduler_base()
+{
+    local base
+
+    if [ -d /usr/libexec/pscheduler ]; then
```

Comment:
Instead of hardcoding paths into the script, use a substitution as done in `list.raw`.  There's a `%: %.raw` target in the Makefile does substitutions; add anything new to that and to the part of the spec that does the build.  RPM macros for the paths are defined in `pscheduler-rpm/pscheduler-rpm.spec`.